apps/ewallet_db/priv/repo/seeds/00_role.exs

```diff
@@ -1,9 +1,10 @@
 defmodule EWalletDB.Repo.Seeds.RoleSeed do
   alias EWalletDB.Role
+  alias EWalletDB.Seeder
```

Comment:
Can be merged with `alias EWalletDB.*` above.

Applies to the rest of the PR as well.